filter/expand_json.go

```diff
@@ -10,11 +10,48 @@ import (
 	"github.com/jmespath/go-jmespath"
 )
 
+const help = `
```

Comment:
rename this to `expandJSONhelp` to not _steal_ the `help` variable name to all code located in the `filter` package